money/tests/views.py

```diff
@@ -15,6 +15,7 @@ class SampleForm(forms.Form):
 class SampleModelForm(forms.ModelForm):
     class Meta:
         model = SimpleMoneyModel
+        fields = '__all__'
```

Comment:
this is just test code but removing the `fields` field just underneath might be a good idea